datasets/fr/assemblee/fr_assemblee.yml

```diff
@@ -0,0 +1,54 @@
+title: Members of the French National Assembly
+prefix: fr-assemblee
+url: https://www.assemblee-nationale.fr/dyn/vos-deputes
+coverage:
+  frequency: daily
+  start: 2024-03-05
+entry_point: crawler.py
+summary: >-
+  Members of the French National Assembly.
+description: |
+  This dataset lists the serving members of the French National Assembly.
+
+  The data is collected from the open data platform for the National
+  Assembly (Assemblée Nationale), which is the lower legislative
+  chamber of the French government.  The National Assembly is composed
+  of 577 members (députés) elected to five-year terms.
+
+  In contrast to the Senate, which represents the local and regional
+  governments (collectivités), the National Assembly represents the
+  people, and is elected by popular suffrage, by a two-round system in
+  single-member constituencies.  The number of constituencies is
+  proportional to the population, with the exception that each
+  département must have at least two representatives, and
+  constituencies may not cross departmental boundaries.
+
+  The function of the Parliament (including the Senate and the
+  National Assembly) is to examine and pass laws and to verify and
+  evaluate public policy and the functioning of the government.
+publisher:
+  name: data.assemblee-nationale.fr
+  description: |
+    The National Assembly maintains an open data site publishing
+    information about its membership and functioning (including
+    committees, meetings, sessions, and votes), in XML, JSON, and CSV
+    format.
+
+    Data is provided under the French Government's [Open License](https://www.etalab.gouv.fr/wp-content/uploads/2017/04/ETALAB-Licence-Ouverte-v2.0.pdf)
+  country: fr
+  url: https://data.assemblee-nationale.fr/
+  official: true
+data:
+  url: https://data.assemblee-nationale.fr/static/openData/repository/16/amo/deputes_actifs_mandats_actifs_organes/AMO10_deputes_actifs_mandats_actifs_organes.json.zip
```

Comment:
Okay, that's probably the one that we want to use to get all the MNAs, similarly to what we'll do for the Senate.

Actually the data URL looks suspiciously ephemeral (with the /16/ in it) so perhaps it would be best to start from the page itself to find the appropriate download link?

Reply:
Also, this file is quite large (11MB) and downloading it fails with incomplete data at least half the time - not sure if that's a problem with my Internet connection, a bug in the `Context` code, or something else :(

Reply:
Ah, it seems to work now.  I won't bother trying to parse the HTML page to get the link for now, since they're just as likely to change the layout as they are to change the download URL.